python/mxnet/ndarray/numpy/_op.py

```diff
@@ -732,3 +733,66 @@ def get_list(arrays):
 
     arrays = get_list(arrays)
     return _npi.stack(*arrays, axis=axis, out=out)
+
+@set_module('mxnet.ndarray.numpy')
```

Comment:
Move this up to after line 350, after `power`, remember the 2-blank-line gap between Python functions.

Reply:
Fixed. Thx!